phivk/gradu.al
# Feature/add data viz for coee

This adds the data visualisation to the COEE page.

To test:
- Check the visualisation on `/coee`
- Delete the edges.json, node.json and popular.json from the content/coee/data directory
- Run `node build-script` from the root directory
  - Confirm the data files have been recreated without errors
  - Confirm the visualisation still works
